src/Modules/Station/Components/StationBrowser/StationBrowser.tsx

```diff
@@ -0,0 +1 @@
+export { StationBrowserForHome } from './StationBrowserForHome';
```

Comment:
Why don't you write code here instead of export from another file in the same folder?

Reply:
We should use StationBrowser instead of StationBrowserForHome